### PR TITLE
Document jsonb_pretty, jsonb_object, and regexp_split_to_array

### LINKED ISSUE #1383: Document: feat(expr): add `regexp_split_to_array`
This issue tracks the documentation update needed for the merged PR #12844.

Source PR URL: https://github.com/risingwavelabs/risingwave/pull/12844
Source PR Merged At: 2023-10-17T03:32:23Z

### LINKED ISSUE #1428: Document: feat(expr): add `jsonb_pretty` function
This issue tracks the documentation update needed for the merged PR #13050.

Source PR URL: https://github.com/risingwavelabs/risingwave/pull/13050
Source PR Merged At: 2023-10-27T08:36:11Z

### LINKED ISSUE #1431: Document: feat(expr): add `jsonb_object` function
This issue tracks the documentation update needed for the merged PR #13036.

Source PR URL: https://github.com/risingwavelabs/risingwave/pull/13036
Source PR Merged At: 2023-10-27T10:31:58Z

### PR DESCRIPTION
**Pull request by neverchanje**
## Info

- **Description**

  - [ What's changed? Which parts of the docs are affected? ]

- **Notes**

  - [ Include any supplementary context or references here. ]

- **Related code PR**

  https://github.com/risingwavelabs/risingwave/pull/13050
  https://github.com/risingwavelabs/risingwave/pull/13036
  https://github.com/risingwavelabs/risingwave/pull/12844

- **Related doc issue**
  
  Resolves https://github.com/risingwavelabs/risingwave-docs/issues/1428 
  Resolves https://github.com/risingwavelabs/risingwave-docs/issues/1431 
  Resolves https://github.com/risingwavelabs/risingwave-docs/issues/1383

## For reviewers

- **Preview**

  - [ Paste the preview link to the updated page(s) here. Edit this item after the preview site is ready. To find the updated pages, scroll down to locate and open the Amplify preview link and select the **dev** version of the documentation. ]

- **Key points**

  - [ Parts that may need revision or extra consideration. ]

## Before merging

- [x] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`CharlieSYH`, `emile-00`, & `hengm3467`).

### CODE REVIEW

File: docs/sql/functions-operators/sql-function-json.md [hunk 1]
```diff
@@ -117,6 +117,37 @@ number
 
 ```
 
+### `jsonb_pretty`
+
+This function takes a `jsonb` object and returns a text representing the formatted, indented JSON value.
+
+```sql title=Syntax
+jsonb_pretty ( jsonb_object JSONB ) → TEXT
+```
```
Comment (wangrunji0408) [inline docs/sql/functions-operators/sql-function-json.md]:
```suggestion
This function takes a `jsonb` value and returns a text representing the formatted, indented JSON value.

```sql title=Syntax
jsonb_pretty ( jsonb JSONB ) → TEXT
```
```

The argument can be any valid json value.

Comment (neverchanje) [reply docs/sql/functions-operators/sql-function-json.md]: Hi Runji,
I see no difference. As long as the value can be parsed into a JSONB, "The argument" must have been confirmed valid.

Comment (wangrunji0408) [reply docs/sql/functions-operators/sql-function-json.md]: Oh I mean types other than objects, like arrays and scalars, can also be the input.

Comment (neverchanje) [reply docs/sql/functions-operators/sql-function-json.md]: Oh, Thanks. I omitted that.

**Review by wangrunji0408 — APPROVED** [review]

**Review by hengm3467 — APPROVED** [review]

**Pull request merged**